Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -74,7 +74,97 @@ TEST_CASE("tag atoms in SVG", "[drawing][SVG]") {
     CHECK(text.find("atom-selector") != std::string::npos);
     CHECK(text.find("bond-selector") != std::string::npos);
   }
+  SECTION("inject prop to class") {
+    auto m1 = "C1N[C@@H]2OCC12"_smiles;
+    REQUIRE(m1);
+
+    for (auto it = m1->beginAtoms(); it != m1->endAtoms(); it++) {
+      auto prop = boost::format("__prop_class_atom_%d") % (*it)->getIdx();
+      (*it)->setProp("_tagClass", prop.str());
+    }
+    for (auto it = m1->beginBonds(); it != m1->endBonds(); it++) {
+      auto prop = boost::format("__prop_class_bond_%d") % (*it)->getIdx();
+      (*it)->setProp("_tagClass", prop.str());
+    }
+
+    MolDraw2DSVG drawer(200, 200, -1, -1, NO_FREETYPE);
+    MolDraw2DUtils::prepareMolForDrawing(*m1);
+    drawer.drawMolecule(*m1);
+    drawer.tagAtoms(*m1);
+    drawer.finishDrawing();
+    std::string text = drawer.getDrawingText();
+    std::ofstream outs("testAtomTags_2.svg");
+    outs << text;
+    outs.flush();
+
+    size_t i = 0;
+    size_t c = 0;
+    while (true) {
+      auto i2 = text.find("__prop_class_atom_", i);
+      if (i2 == std::string::npos) break;
```

Comment:
I know it seems like a picky thing, but this is something we try to be consistent about: 
```suggestion
      if (i2 == std::string::npos) {
         break;
      }
```

Reply:
Changed